src/types/instantsearch.ts

```diff
@@ -112,6 +113,10 @@ export type StateMapping<TRouteState = UiState> = {
 export type Client = AlgoliaSearchClient;
 
 export type Helper = AlgoliaSearchHelper;
```

Comment:
wouldn't you want to modify the type of Helper.derive to return this DerivedHelper type then?

Reply:
It's useless, the current definition uses `any`. It bypass any other definition.